fsm/okta/inactive_users/README.md

```diff
@@ -0,0 +1,36 @@
+## Inactive Users - Okta
+
+**As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account.**
+
+### What it does
+
+This Policy Template leverages the [Okta Users API](https://developer.okta.com/docs/reference/api/users/#list-users) to identify users that have no logged in for an extended period of time, and optionally will deactivate those users upon approval.  
+
+### Prerequesites
+
+- This Policy Template requires an Okta API Token to authenticate with the Okta API.  [This tutorial](https://developer.okta.com/docs/guides/create-an-api-token/overview/) describes how to create a new Token.
+- The Okta API Token must then be stored as a RightScale Credential in the account in which this policy will be applied. The credential must be named `OKTA_API_KEY`.
+
```

Comment:
Any special permissions needed in Okta, or as long as you can create an API Token you can manage users?

Reply:
You have to be an admin to access that portal.  fyi it says so in that docs link